setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
```

Comment:
have you verified it yourself on clean install?

Reply:
not on my system, but the github action runs from a clean system and it seems to work fine.